LICENSE.md

```diff
@@ -1789,3 +1789,49 @@ The original license notice is included below:
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 ```
+
+# License notice for  re2s regular expression code
```

Comment:
Please delete a section called `License notice for marcomaggi/cre2` above. We don't use cre2 so it's not needed any longer.

Reply:
Done.

 I was not sure if it needed to be retained because both the original SN devo
and I looked at and benefited from that code. In my prior practice, that would
have 'tainted' us.  Thank you for the umpires call, which, as always, if final.